vc-worker.py

```diff
@@ -50,10 +50,9 @@ def process_deploy_action(conn, action, vc):
 
         template = get_template(machine_ro.labels)
 
-        if Settings.app['vsphere']['default_network_name']:
+        network_interface = get_network_interface(machine_ro.labels)
```

Comment:
I would suggest to move this to else branch for the if below.

Reply:
+1 from me, it will be more efficient and change even more readable :)